components/edit-collective/EditTiers.js

```diff
@@ -410,6 +415,14 @@ class EditTiers extends React.Component {
 
         <div className="tiers">
           <h2>{this.props.title}</h2>
+          <Flex flexDirection="column" alignItems="center" justifyContent="center" minWidth={300}>
+            <StyledCheckbox
+              name="custom-contributions"
+              label={intl.formatMessage(this.messages['customContributions.label'])}
+              defaultChecked={defaultIsChecked}
+              width="auto"
```

Comment:
How should I edit the setting ? should I use `updateSettingsMutation` from './mutations' ?